plugin/src/test/groovy/com/novoda/staticanalysis/internal/ktlint/KtlintIntegrationTest.groovy

```diff
@@ -124,4 +137,12 @@ class KtlintIntegrationTest {
                     maxErrors = ${maxErrors}
                 }""")
     }
+
+    private String getDefaultConfig() {
```

Comment:
Just an optional suggestion: 

Since this test is already parameterized, this `defaultConfig` can also be another property on the parameters. Thinking about this also because of `>=` on the `String` type below.

Reply:
I like that a lot better. Will make any future changes to the config feel much cleaner too.

Reply:
Awesome. Thanks